### PR TITLE
Fix for ReactNative 0.25+

### PR DESCRIPTION
**Pull request by alimek**
Added React as a dev dep and changed to use component from react rather than react-native

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 
 const {
   PanResponder,
   View,
   TouchableWithoutFeedback,
+} = ReactNative;
+
+const {
   Component,
 } = React;
```
Comment (grabbou) [inline index.js]: Can we replace `extends Component` with `React.Component` ? I am pretty sure it's not that needed to destruct it here, @Kureev ?

**Comment by grabbou** [comment]
Thanks! Just waiting for @Kureev thoughts and let's shipit 🚢

**Comment by alivelee** [comment]
And in the example the <Menu /> also is transpiled to `ReactNative.createElement` calls.

**Comment by Kureev** [comment]
@grabbou :shipit: if it looks good to you

**Comment by Kureev** [comment]
Published in `0.19`

**Pull request merged**